### PR TITLE
vfc: Correct spelling typos

### PR DESCRIPTION
**Pull request by abdhaleegit**
Corrected few spelling mistakes in log.info
messages

Signed-off-by: Abdul Haleem <abdhalee@linux.vnet.ibm.com>

### CODE REVIEW

File: io/disk/vfc-tests/virtual_fc.py [hunk 1]
```diff
@@ -351,7 +351,7 @@ def is_path_online():
                                    " % (operation, self.path, path_stat))
                     return True
             else:
-                self.log.info("Operation unknown, provide correct opertion")
+                self.log.info("Operation unknown, provide correct operation")
```
Comment (Naresh-ibm) [inline io/disk/vfc-tests/virtual_fc.py:354]: @abdhaleegit these are intentionally made once, so that line length is <79 character.

Comment (Naresh-ibm) [reply io/disk/vfc-tests/virtual_fc.py:354]: any way Thanks for changing them to proper spells.

File: io/disk/vfc-tests/vfc_dlpar.py [hunk 2]
```diff
@@ -289,9 +289,9 @@ def mpath_verification(self, operation, paths, drc):
                     err_paths.append(path)
 
         if err_paths:
-            self.fail("path verfction failed for drc %s:%s" % (drc, err_paths))
+            self.fail("path verification failed for drc %s:%s" % (drc, err_paths))
         else:
-            self.log.info("path verfction success for drc :%s" % drc)
+            self.log.info("path verification success for drc :%s" % drc)
```
Comment (Naresh-ibm) [inline io/disk/vfc-tests/vfc_dlpar.py:294]: @abdhaleegit please check the pep8 once,
```

[root@server ~]# pycodestyle test.py 
test.py:252:80: E501 line too long (81 > 79 characters)
test.py:254:80: E501 line too long (83 > 79 characters)
test.py:270:80: E501 line too long (81 > 79 characters)
test.py:272:80: E501 line too long (84 > 79 characters)
test.py:292:80: E501 line too long (82 > 79 characters)
[root@server ~]# 
```

**Review by Naresh-ibm — CHANGES_REQUESTED** [review]

File: io/disk/vfc-tests/virtual_fc.py (hunk 1 above)
Comment (Naresh-ibm) [reply io/disk/vfc-tests/virtual_fc.py:354]: need the pep8 changes here too,
```

[root@server ~]# pycodestyle vfc.py 
vfc.py:286:80: E501 line too long (80 > 79 characters)
vfc.py:301:80: E501 line too long (80 > 79 characters)
[root@server ~]#
```

**Review by Naresh-ibm — CHANGES_REQUESTED** [review]

File: io/disk/vfc-tests/virtual_fc.py (hunk 1 above)
Comment (abdhaleegit) [reply io/disk/vfc-tests/virtual_fc.py:354]: if lines go beyond 79 chars.. than better to make it in 2 lines, instead triming words :-)

Comment (abdhaleegit) [reply io/disk/vfc-tests/virtual_fc.py:355]: done

File: io/disk/vfc-tests/vfc_dlpar.py [hunk 3]
```diff
@@ -222,7 +222,7 @@ def device_add_remove(self, operation, vfc_dic):
                       vfc_dic["r_lpar"], vfc_dic["r_slot"], vfc_dic["wwpn"])
         else:
             cmd = 'chhwres -r virtualio -m %s -o r -p %s  -s %s' \
-                   % (self.server, self.lpar, vfc_dic["c_slot"])
+                % (self.server, self.lpar, vfc_dic["c_slot"])
```
Comment (Naresh-ibm) [inline io/disk/vfc-tests/vfc_dlpar.py:225]: @abdhaleegit looks like some indentation here. please re-check as this a command ad it may fail on script run

Comment (abdhaleegit) [reply io/disk/vfc-tests/vfc_dlpar.py:225]: Not a problem I see when running the test

**Review by Naresh-ibm — APPROVED** [review]
LGTM

**Pull request merged**